### PR TITLE
Working on adding test coverage monitoring

### PR DESCRIPTION
**Pull request by aaron-prindle**
#490

### CODE REVIEW

File: test.sh [hunk 1]
```diff
@@ -25,11 +25,17 @@ else
...
-# Run "go test" on packages that have test files.
+
+# Run "go test" on packages that have test files.  Also create coverage profile
 echo "Running go tests..."
 cd ${GOPATH}/src/${REPO_PATH}
-TESTS=$(go list -f '{{ if .TestGoFiles }} {{.ImportPath}} {{end}}' ./...)
-go test -v ${TESTS}
+rm -f out/coverage.txt
+echo "mode: count" > out/coverage.txt
+for pkg in $(go list -f '{{ if .TestGoFiles }} {{.ImportPath}} {{end}}' ./...); do
```
Comment (dlorenc) [inline test.sh]: Could you add some comments explaining how this works? The tail -n +2 specifically.

It also might be better to move the coverage stuff to a separate script, and give it a "make coverage" command, then invoke that from Travis as well. WDYT?

Comment (aaron-prindle) [inline test.sh]: Sure, I can definitely add comments to this.  The reason that I added the coverage generation to the tests is that I believe that generating test coverage and testing are coupled.  The only way I know to generate coverage profiles is through go test.  I believe having a separate `make coverage` would require rerunning all of the tests in travis.

Comment (dlorenc) [inline test.sh]: Ah right. Keeping them separate SGTM then. We can split it up if it slows down the tests too much for normal development.

**Comment by dlorenc** [comment]
LGTM

**Pull request merged**